cdap-common/src/main/java/co/cask/cdap/gateway/handlers/ConfigHandler.java

```diff
@@ -97,15 +98,60 @@ public void configHBase(@SuppressWarnings("UnusedParameters") HttpRequest reques
     }
   }
 
-  private ChannelBuffer stringWriter2ChannelBuffer(StringWriter stringWriter) {
+  private ChannelBuffer newChannelBuffer(StringWriter stringWriter) {
     return ChannelBuffers.copiedBuffer(ByteOrder.BIG_ENDIAN, stringWriter.toString(), Charsets.UTF_8);
   }
 
-  private Map<String, String> toMap(Iterable<Map.Entry<String, String>> configuration) {
-    ImmutableMap.Builder<String, String> result = ImmutableMap.builder();
+  private String getFirstElement(String[] array) {
+    if (array != null && array.length >= 1) {
+      return array[0];
+    } else {
+      return null;
+    }
+  }
+
+  private List<ConfigEntry> toObject(Configuration configuration) {
```

Comment:
Rather than super generic name `toObject`, better call it `toConfigEntries` or `toConfigEntryList`.